python/tvm/relay/frontend/tensorflow.py

```diff
@@ -788,6 +788,14 @@ def _impl(inputs, attr, params, mod):
     return _impl
 
 
+def _expm1():
```

Comment:
Could you add the docstring  describing what this operation is doing?

Reply:
Okay, I added TF description address